packages/web/src/scss/components/Navigation/README.md

```diff
@@ -0,0 +1,82 @@
+# Navigation
+
+The `Navigation` component is a container for the navigation links of the application.
+
+It consists of a these parts:
+
+- [Navigation](#navigation)
+- [NavigationLink](#navigationlink)
+
+## Navigation
+
+The `Navigation` is a `nav` wrapper for lists of links or other navigation components.
+
+```html
+<nav class="Navigation">
+  <ul>
+    <li>
+      <!-- link -->
+    </li>
+  </ul>
```

Comment:
Do I need the list or not?

Reply:
You don't, but it will be more often like this. I added a note to the docs.